extmod/modframebuf.c

```diff
@@ -115,7 +171,7 @@ static inline uint32_t getpixel(const mp_obj_framebuf_t *fb, int x, int y) {
 }
 
 STATIC void fill_rect(const mp_obj_framebuf_t *fb, int x, int y, int w, int h, uint32_t col) {
-    if (x + w <= 0 || y + h <= 0 || y >= fb->height || x >= fb->width) {
```

Comment:
I have a question about this part. What was a purpose of the part that I have changed? As for me it was pointless check but I could be wrong. So let's discuss this